New log file opened at Mon Nov 14 14:38:26 CET 2016 by 3740@Jessica-PC
Taking next connection to: 61214
New client attempting to connect on port: 61215-61214
  Address = /10.0.2.15:61214
This client on port 61215 has uuid 096c1aa4-e0d4-41e1-96dd-f5dc72581611 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-096c1aa4-e0d4-41e1-96dd-f5dc72581611
Taking next connection to: 61214
New client attempting to connect on port: 61256-61214
  Address = /10.0.2.15:61214
Handshake exception on socket: 61256
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 61214
New client attempting to connect on port: 61257-61214
  Address = /10.0.2.15:61214
This client on port 61257 has uuid c7c91f96-4da7-45ee-a479-bc75841d7ed7 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-096c1aa4-e0d4-41e1-96dd-f5dc72581611, play-fork-run-c7c91f96-4da7-45ee-a479-bc75841d7ed7
Taking next connection to: 61214
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 61214
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 61214
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 61214
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 61214
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 61214
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 61214
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 61214
Checking to see if clients are empty...
No non-daemon clients connected after 3 min. Shutting down.
1 daemon clients are connected.
Server socket thread exiting.
1 clients open, will close them...
All client sockets have been closed.
Closing the 3740@Jessica-PC logs at Mon Nov 14 15:04:47 CET 2016, goodbye.
